internal/target/repository_tcp_target.go

```diff
@@ -108,9 +108,9 @@ func (r *Repository) CreateTcpTarget(ctx context.Context, target *TcpTarget, opt
 // UpdateTcpTarget will update a target in the repository and return the written
 // target. fieldMaskPaths provides field_mask.proto paths for fields that should
 // be updated.  Fields will be set to NULL if the field is a zero value and
-// included in fieldMask. Name and Description are the only updatable fields,
-// If no updatable fields are included in the fieldMaskPaths, then an error is
-// returned.
+// included in fieldMask. Name, Description, and WorkerFilter are the only
```

Comment:
I'm not sure that WorkerFilter is updatable in this PR. I think, you need to modify the "immutable_columns" trigger on the target_tcp table.  Also, I'm not sure there's any tests to confirm that WorkerFilter is updatable.

Reply:
I'm not following this...there's a test that adds a filter and then gets authorized sessions and ensures that we see the correct workers as the filter changes. It's not immutable (nor should it be). What makes you think this doesn't work?

Reply:
Added another case in the test to verify the update case.